src/lib/viewers/BaseViewer.js

```diff
@@ -153,6 +154,10 @@ class BaseViewer extends EventEmitter {
 
         this.annotationControlsFSM = new AnnotationControlsFSM();
 
+        if (this.options.showAnnotationsDrawingCreate) {
```

Comment:
Can we always instantiate the module to avoid existence checks every time we want to use it?